.github/workflows/_build.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: _build
+
+on:
+  workflow_call:
+    inputs:
+      node_version:
+        description: The Node.js version.
+        type: string
+        required: false
+        default: "18"
+    outputs:
+      artifact_name:
+        description: The artifact name.
+        value: build-${{ github.sha }}
+
+jobs:
+  build:
+    name: Package
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          node_version: ${{ inputs.node_version }}
+          node_auth_token: ${{ secrets.GH_TOKEN }}
+      - name: Build
+        run: npm run build
+      - name: Package
+        run: npm pack
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-${{ github.sha }}
+          if-no-files-found: error
+          path: "*.tgz"
```

Comment:
I can't remember if we had an offline convo about this but I mentioned in the PR on seamapi/fake-template that a reusable action might be easier here, don't think it ever got a response

https://github.com/seamapi/fake-template/pull/12#discussion_r1176688133

Reply:
This is something I [implemented a while ago](https://github.com/makenew/jsmodule/commits/af556f2a478d36382b198cce97a82d5867154bd1/.github/workflows/build.yml?browsing_rename_history=true&new_path=.github/workflows/_build.yml&original_branch=main), and I can't remember off the top of my head why I made this a workflow vs. an action. I am ok making it an action, but I'd need to look into why I didn't do that in the first place. I'll need to do some digging and get back to you.

Reply:
At this moment, I do not have the bandwidth to convert this to an action. If this is blocking you can open an alternative PR with the change (or a smaller one with just the other actions). Or if this can wait I can test some alternatives out upstream.

Reply:
not blocking, just would be nice to simplify in the future if possible :)